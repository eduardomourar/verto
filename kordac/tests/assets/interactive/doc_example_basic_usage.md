{interactive name="binary_search" type="in-page"}
