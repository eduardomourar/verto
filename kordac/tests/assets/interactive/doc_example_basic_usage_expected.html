<div>
 {{% include 'interactive/binary_search/index.html' %}}
</div>
